staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -78,6 +78,7 @@ var baseOpts = []VersionedOptions{
 		EnvOptions: []cel.EnvOption{
 			cel.CrossTypeNumericComparisons(true),
 			cel.OptionalTypes(),
+			library.Quantity(),
```

Comment:
We should add a 1.29 section and include Quantity there. We can also set DefaultCompatibilityVersion to 1.28.  Any test cases or features that need this feature immediately (e.g. ValidatingAdmissionPolicy v1beta1) should use 1.29 compatibility instead of DefaultCompatibilityVersion.

Reply:
Does it not go into 1.28 compatibility because it was not available in 1.27 so would not be able to rollback?

Reply:
Oh I messed this up. Apologies. It does go in 1.28.  I had forgotten we had already added a new version in this cycle.  You had this right originally-- `DefaultCompatibilityVersion` should be 1.27 and your addition should be for 1.28.

Reply:
I added it back to 1.28 and changed `DefaultCompatibilityVersion ` back to 1.27